public/leie.html

```diff
@@ -52,6 +52,15 @@
       <p>Escape har en kapasitet på 200 personer, inkludert funksjonærer.</p>
       <h3>Når er Escape ledig?</h3>
       <p>Det varierer veldig; alt vi vet er at Escape <em>ikke</em> er ledig på tirsdager innad i semesteret. Utenom tirsdager, <a href="mailto:utlan@cyb.no" title="Send en e-post til utlånsansvarlig">send oss en e-post med ønsket tidspunkt</a> så tidlig som mulig.</p>
+      <h3>Hva slags teknisk utstyr er tilgjengelig?</h3>
+      <p>Escape har en god del teknisk utstyr til disposisjon. Blandt annet:</p>
+      <ul>
+        <li>Mikrofoner (Håndholdt og hodebøyle)</li>
+        <li>Prosjektor med en rekke utganger</li>
+        <li>DJ Utstyr (CDJ2000NXS og DJM2000)</li>
+        <li>Top moderne lysrigg</li>
+      </ul>
+      <p>Send gjerne mail til <a href="mailto:tekniskansvarlig@cyb.no" title="Send en e-post til teknisk ansvarlig">teknisk ansvarlig</a> om det er noe du lurer på!</p>
```

Comment:
Kanskje ikke? Det som fikk meg til å skrive dette var at jeg fikk en mail fra PsyDoc for to dager siden som jeg ikke hadde lagt merke til hvor de hadde sendt mail om å leie til RF Regi og CCa tekans mailen.